themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties

```diff
@@ -88,7 +88,7 @@ offlineToken=Offline Token
 revoke=Vergunning intrekken
 configureAuthenticators=Ingestelde authenticators
 mobile=Mobiel nummer
-totpStep1=Installeer <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> of Google Authenticator op uw apparaat. Beide toepassingen zijn beschikbaar in <a href="https://play.google.com">Google Play</a> en de Apple App Store.
+totpStep1=Installeer een van de onderstaande applicaties op Uw mobiele apparaat
```

Comment:
In other places in this translations file the formal is written without capitalization, would recommend to keep this consistent. So `uw` instead of `Uw`.